cvat/apps/engine/serializers.py

```diff
@@ -591,6 +592,7 @@ class JobReadSerializer(serializers.ModelSerializer):
     stop_frame = serializers.ReadOnlyField(source="segment.stop_frame")
     frame_count = serializers.ReadOnlyField(source="segment.frame_count")
     assignee = BasicUserSerializer(allow_null=True, read_only=True)
+    assignee_updated_date = serializers.DateTimeField(allow_null=True, required=False)
```

Comment:
It looks like, you can skip defining assignee_updated_date here since you've added it in the model.

Reply:
Updated